backend/src/app/features/components_v2.clj

```diff
@@ -227,8 +227,8 @@
                     [shape]
                     (if (or (nil? (:parent-id shape)) (ctk/instance-head? shape))
                       (assoc shape
-                             :type :frame           ; Old groups must be converted
-                             :fills []              ; to frames and conform to spec
+                             :type :frame                 ; Old groups must be converted
+                             :fills (or (:fills shape)[]) ; to frames and conform to spec
                              :hide-in-viewer true
                              :rx 0
                              :ry 0)
```

Comment:
I'd rather use a `(cond->` to set each attribute only if the value is null in the original shape. Think that if the shape was already a frame in v1, it may have values in those attributes, that we need to preserve.

Reply:
great!